tests/API_contract_tests.postman_collection.json

```diff
@@ -1656,20 +1654,20 @@
 						{
 							"listen": "prerequest",
 							"script": {
-								"id": "d7808f80-2e6d-4863-bbed-58bc063a59f2",
+								"id": "b906c075-b943-4d3f-b8cf-a583ca993ed6",
 								"exec": [
-									"const moment = require('moment');",
-									"pm.variables.set(\"currentTimestamp\", moment().unix());",
 									"const uuid = require('uuid');",
-									"pm.variables.set(\"signupPw\", uuid.v4().slice(0,9));"
+									"pm.variables.set(\"signupPw\", uuid.v4().slice(0,9));",
+									"let dataUserEmailAddress = \"automated-tests+user\"+ Date.now() + \"@tidepool.org\";",
```

Comment:
I think these, email addresses, should come from travis config rather than checked into code

Reply:
see https://github.com/tidepool-org/APITests#cli section where we pass in emails from config

Reply:
**login email declared in environment:**
https://github.com/tidepool-org/APITests/blob/master/tests/QA1.postman_environment.json#L11

**login email variable used in tests:**
https://github.com/tidepool-org/APITests/blob/master/tests/API_contract_tests.postman_collection.json#L35

**and then travis config declares variable that is pulled from config**
https://travis-ci.com/github/tidepool-org/APITests/builds/190372489/config

**was:**
`\"automated-tests+user\"+ Date.now() + \"@tidepool.org\";"`

**could be in config:**
`automated-tests+user_datetag_@tidepool.org`

**then converts in code:**
`let uniqeEmail = {{fromConfig}}.replace('_datetag_', Date.now())`

Reply:
thanks jamie. great suggestion. Also, did that for the sharing email address we have in the smoke test